assets/src/sass/blocks/_button.scss

```diff
@@ -12,20 +12,6 @@
 	}
 }
 
-@mixin button-icon-rules {
-	background-image: url("./assets/src/svg/lock-white.svg");
-	background-position: center;
-	background-repeat: no-repeat;
-	background-size: contain;
-	content: "";
-	display: inline-block;
-	height: 1em;
-	margin-right: 0.5em;
-	position: relative;
-	top: 2px;
-	width: 1em;
-}
-
```

Comment:
I moved it from here to `_mixins-buttons.scss` as you'll see below, so it can be reused.